WordPress/Classes/ViewRelated/Site Creation/Final Assembly/SiteAssemblyWizardContent.swift

```diff
@@ -268,6 +268,7 @@ private extension SiteAssemblyWizardContent {
 
     func dismissTapped(viaDone: Bool = false, completion: (() -> Void)? = nil) {
         // TODO : using viaDone, capture analytics event via #10335
+        RootViewCoordinator.shared.isSiteCreationActive = false
```

Comment:
I added this as a safety measure. I left the one below in place regardless as we need to set the state before reloading the UI.